### PR TITLE
[New] Add unistore@3.0.3 w/ npm auto-update

### PR DESCRIPTION
**Pull request by kirintwn**
Pull request for issue: #12323 

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/developit/unistore
 * Official website (optional, not the repository):
 * NPM package url (optional): https://www.npmjs.com/package/unistore
 * License and its reference: MIT
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 29
   - Count of stars: 1262
   - Count of forks: 30
 * NPM download stats (optional):
   - Downloads in the last day: 0
   - Downloads in the last week: 352
   - Downloads in the last month: 529

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 800 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/unistore/package.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "unistore",
+  "filename": "dist/unistore.umd.js",
+  "version": "3.0.2",
+  "description": "Dead simple centralized state container (store) with preact and react bindings.",
+  "keywords": [
+    "preact",
+    "component",
+    "state machine",
+    "redux"
```
Comment (sufuf3) [inline ajax/libs/unistore/package.json]: Let's add more keyword, let developers can easier find this library.

Thanks

**Comment by Kanaye** [comment]
@extend1994 you are totally right only the `*.umd.js` files use umd as the module format.
All files not containing `umd` use es modules which isn't supported in most browsers (today) and are indeed intended for bundlers supporting es-modules (e.g. webpack or rollup).

**Comment by developit** [comment]
I'd recommend hosting both, since people are slowly starting to use ES Modules in browsers.  An example of the UMD files is here:
https://unpkg.com/unistore@3.0.5/full/preact.umd.js

**Comment by sashberd** [comment]
@extend1994 @PeterDaveHello So what is your conclusions about this PR?

**Comment by PeterDaveHello** [comment]
@extend1994 usually not, but if the upstream provides minified es files, and make sure to let their users know how to  properly reference to those minified resources, we are good to host them.

**Comment by extend1994** [comment]
OK!

**Pull request merged**